tools/build.sh

```diff
@@ -4,12 +4,20 @@ if [[ -f ".circleci/debuglog" ]]; then
 	set -x
 fi
 
-# Check if there is a build.sh in the repo and exec
 if [[ -x "tools/build.sh" ]]; then
 	tools/build.sh
 	exit $?
 fi
 
+if [[ -f "glide.lock" ]]; then
+
+    if [[ ! -d "vendor" ]]; then
+        echo "VENDOR DOT SH"
+        tools/vendor.sh
```

Comment:
@andrewwatson I don't think we actually want to run tools/vendor.sh since that is typically 
`glide cc && glide up && glide install --strip-vendor` which would potentially bring in new versions with every build.

I think we just want to run `glide install --stripe-vendor`

Reply:
ah, yes.